pkg/reconciler/core/logicalclusterdeletion/deletion/logicalcluster_resource_deletor_test.go

```diff
@@ -59,15 +72,19 @@ func init() {
 			},
 			gvrError:            fmt.Errorf("test error"),
 			expectErrorOnDelete: fmt.Errorf("test error"),
-			expectConditions: conditionsv1alpha1.Conditions{
-				{
-					Type:   tenancyv1alpha1.WorkspaceDeletionContentSuccess,
-					Status: v1.ConditionFalse,
-				},
-				{
-					Type:   tenancyv1alpha1.WorkspaceContentDeleted,
-					Status: v1.ConditionTrue,
-				},
+			expectConditions: conditionsv1alpha1.Conditions{{
+				Type: tenancyv1alpha1.WorkspaceContentDeleted,
+				// TODO (fgiloux): to be confirmed:
+				// this was previously set to true
+				// and WorkspaceDeletionContentSuccess was set to false
+				// WorkspaceDeletionContentSuccess does not exist anymore
+				// setting WorkspaceContentDeleted to false in case of
+				// error during discovery seems right
+				// looking at the code it seems that an error may be
+				// generated when the discovery client cannot be instantiated
+				// or when the preferred resource cannot be found
```

Comment:
Could you confirm that the assumption is correct?

Reply:
I think yes.

Reply:
My impression was that the second condition was not really clear. Didn't really understand it.